pypowsybl/network/impl/network_creation_util.py

```diff
@@ -152,6 +152,14 @@ def create_micro_grid_nl_network() -> Network:
     """
     return _create_network('micro_grid_nl')
 
+def create_metrix_tutorial_six_buses_network() -> Network:
```

Comment:
you need to add the method to network/impl/__init__ to access it outside the intern module 
I do it